🔥Bug Hotfix & Visual Improvements

Algamja11 opened the pull request:
# Bug Fixes
|Slot Rendering|Waypoint Screen|
|--|--|
|<img src="https://github.com/user-attachments/assets/ce643b6a-cddb-48c7-94db-d1d52c940029" width="300" height="300"/>|<img src="https://github.com/user-attachments/assets/385785df-e682-4eb2-afad-fcfcfa9a34b2" width="300" height="300"/>|
|Slot items are now clipped.|Waypoint highlighting now works properly|
|Makes compatibility with mods that modify slot rendering.|Changed waypoint highlighting method|
|(List rendering independent from GuiSlotMinimap.)|(Double click waypoint activation button -> Click waypoint icon)|

# Updates (Visual Improvements)
|Add Waypoint Screen / Color Picker|Add Waypoint Screen / Icon Picker|Mob Selection (sorting and indicator)|
|--|--|--|
|<img src="https://github.com/user-attachments/assets/ecadf4c2-8277-4450-aaa7-85150c0596d0" width="400" height="300">|<img src="https://github.com/user-attachments/assets/1345fb47-3cf4-4c79-a1ac-18214bc3a1ef" width="400" height="300">|<img src="https://github.com/user-attachments/assets/23e4adac-deb7-45dd-831f-afbf692e6e9c" width="400" height="300">|
|RGB values are shown.|The selected icon is highlighted.|Sort by: Passive -> Hostile -> Neutral|

   
   
   

# I'm trying to improve VoxelMap :)

Brokkonaut commented:
Can you please avoid wildcard imports? Otherwise looks good :)

fantahund commented:
one more thing from me. I like your changes. But can you remove the entity icons from the mob list? i don't think it's pretty.

fantahund commented:
or you try to display the correct entity icons that are also displayed on the minimap. but this could be more complicated.

Brokkonaut commented:
One more issue I found: In the new waypoint screen the scroll bar for the world list ist now moved to the side and far off from the list and it is colored in the waypoint color instead of gray

Algamja11 commented:
> or you try to display the correct entity icons that are also displayed on the minimap. but this could be more complicated.

It's too hard for me but I can remove icon from the list.
I'm not home right now, so I fix it about 4 hours later.

Algamja11 commented:
> One more issue I found: In the new waypoint screen the scroll bar for the world list ist now moved to the side and far off from the list and it is colored in the waypoint color instead of gray

I found how to fix it.
I'll fix this later too

Algamja11 commented:
Done, I fixed syntax problem, color picker scale, and list rendering.
And I fixed the width issue of the dimension list, but I couldn't fix the color issue of the dimension list because I couldn't find the cause.
I'm sorry I can't fix all the problems.

Algamja11 commented:
Oh I found cavemap background issue (#53 )
I know it's not a good idea to commit off-topic, but I'm committing it because I think it's an important change.

fantahund commented:
thx ^^
